VMBackup/main/handle.py

```diff
@@ -165,24 +165,24 @@ def snapshot():
         backup_logger.log(errMsg, False, 'Error')
     snapshot_done = True
 def daemon():
-    global MyPatching,backup_logger,hutil,run_result,run_status,error_msg,freezer,para_parser
+    global MyPatching,backup_logger,hutil,run_result,run_status,error_msg,freezer,para_parser,snapshot_done
```

Comment:
@ahmetalpbalkan added the space. on running pep8 there are lot of spacing issue which we will take up while refactoring the VMBackup extension code